packaging/deb/templates/control

```diff
@@ -19,12 +19,12 @@ Depends: ${misc:Depends}, ${python:Depends},
  python-lmdb (>= 0.87),
  python-setuptools,
  python-wtforms (>= 2.0.1)
-Recommends: torch7-nv
+Recommends: torch7-nv,
+ tensorflow-gpu
```

Comment:
Just remove this for now. I'll make a separate packaging PR later. Without a working TF deb, this DIGITS deb becomes unusable.